src/index.ts

```diff
@@ -248,6 +246,19 @@ task("starknet-compile", "Compiles Starknet contracts")
         }
     });
 
+async function traverseFiles(traversable: string, fileCriteria: string = "*") {
+    let paths: string[] = [];
+    if (fs.lstatSync(traversable).isDirectory()) {
+        const globPromise = promisify(glob);
+        paths = await globPromise(path.join(traversable, "**", fileCriteria));
+    }
+    else {
+        paths.push(traversable);
+    }
+    const files = paths.filter(file => { return fs.lstatSync(file).isFile(); });
```

Comment:
no need for using the `return` statement and curly brackets, this should be valid: `file => fs.lstatSync(file).isFile()`